lib/core-server/src/utils/server-address.ts

```diff
@@ -15,3 +15,7 @@ export const getServerPort = (port: number) =>
     logger.error(error);
     process.exit(-1);
   });
+
+export const getServerChannelUrl = (port: number, { https }: { https?: boolean }) => {
+  return `${https ? 'wss' : 'ws'}://localhost:${port}/`;
```

Comment:
is it always localhost?

Reply:
I followed the pattern on `getServerAddresses` above. I'm not totally sure it is always `localhost` but let's start with that and see.